tests/test_profile_data.py

```diff
@@ -176,3 +176,23 @@ def test_profile_using_spark(df_spark):
     )
 
     assert len(generate_raw_profile_dataset([["first_name", "blank"]], linker)) == 0
+
+
+def test_profile_null_columns(caplog):
```

Comment:
Feel free to try making this backend agnostic, should you have the time.

I believe Afua is transforming the rest into backend agnostic tests, so it would just be this one that needs attention.

Reply:
Done. Forced to ignore Spark unless there's a non-fiddly workaround.